crates/holochain/src/core/ribosome/host_fn/get_link_details.rs

```diff
@@ -147,4 +157,34 @@ pub mod slow_tests {
         }
         assert!(remove_happened);
     }
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn get_link_details_local_only() {
+        holochain_trace::test_run().unwrap();
+        // agents should not pass around data
+        let config = SweetConductorConfig::rendezvous(false).tune(|config| {
+            config.disable_historical_gossip = true;
+            config.disable_recent_gossip = true;
+            config.disable_publish = true;
+        });
+        let mut conductors = SweetConductorBatch::from_config_rendezvous(2, config).await;
+        let (dna_file, _, _) = SweetDnaFile::unique_from_test_wasms(vec![TestWasm::Link]).await;
+        let apps = conductors.setup_app("test", &[dna_file]).await.unwrap();
+
+        // alice creates a link
+        let zome_alice = apps[0].cells()[0].zome(TestWasm::Link.coordinator_zome_name());
+        let _: ActionHash = conductors[0].call(&zome_alice, "create_link", ()).await;
+
+        // now make both agents aware of each other
+        conductors.exchange_peer_info().await;
```

Comment:
This shouldn't be necessary with rendezvous right? Or is that `false` above turning off bootstrap?